tsconfig.base.json

```diff
@@ -15,7 +15,6 @@
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
-    "resolveJsonModule": true,
```

Comment:
question: is this intentional? Wouldn't it break importing the translations?

Reply:
If you look at the whole file, it was added twice